### ISSUE TITLE
Mars Biomes

### ISSUE THREAD

**Issue by ZBok** [KSP-RO/RealSolarSystem#163]
I think the Olympus Mons biome is incorrectly placed. The surface map and biome map seem to be oriented correctly and overlap nicely, but the Olympus Mons biome can be found at several different locations around the planet, similar to Midlands/Lowlands/Highlands.

The actual Olympus Mons location does show the Olympus Mons biome.

**Comment by PhineasFreak** [comment]
Did you validate that via the in-game cheat option or just by luck? Because it is known that biome maps can be buggy at times (in stock KSP you can find Tundra just to the east of KSC).

**Comment by ZBok** [comment]
Hmm I used the SCANsat biome map to check as well as the biome map, it's not small isolated areas like Tundra near KSC, more like really large biome sized areas. I didn't land anywhere to check, though another guy user on reddit mentioned a similar problem yesterday, same Olympus Mons biome just nowhere near the mountain.

**Comment by jonassm** [comment]
Im said guy on reddit! I noticed while doing a fly by(inclined aswell) of Mars in RP-1, that the Olympus Mons biome felt like it was everywhere. (Was in said biome for over 24 hours). Also saw Tyler Raiz who does youtube videos on RO/RP1 having the same thing

**Comment by ZBok** [comment]
Hiya said reddit guy! Is your name actually Jonas?

**Comment by PhineasFreak** [comment]
If the Olympus Mons biome (colored in bright blue) exists in other places then [you will have to open the Mars biome texture](https://github.com/KSP-RO/RSS-Textures/blob/master/8192/PluginData/MarsBiomes.png) and hunt down any other sections (or even single pixels) that have the same color code.

**Comment by leudaimon** [comment]
Could there be a name switch in the config?

**Comment by PhineasFreak** [comment]
Does not look like it: https://github.com/KSP-RO/RealSolarSystem/blob/master/GameData/RealSolarSystem/RSSKopernicus/Mars/Mars.cfg#L45-L107

And the config color value for the Olympus Mons biome (0.071, 0, 1) translates exactly to the color that the biome map uses (18, 0, 255).

I also combed the entire texture pixel by pixel to see if the color repeats somewhere else but no other biome uses the same color code. Maybe KSP gets confused by the other similar colors (cyan, bark blue, magenta, purple)? Should not though...

**Comment by jonassm** [comment]
I sent some probes to Mars today so i got some anecdotal data, which is very reliable! /s (seriously though, prefacing by saying these are not very scientific tests)

I noticed that the Olympus Mons biome might feel like a biome the game just uses when it doesn't know what else to do. Specifically, when i was over Vallas Marineras, and i was going to Midlands, there was about 2 seconds where the game went to Olympus Mons and then Midlands. This was not even only one specific spot, it happened multipile times.

TL;DR game might use olympus mons as a "default" biome when its confused

**Comment by pap1723** [comment]
I noticed something similar when I made biome maps for the Moon and Earth a long time ago. I found that there are safe colors to use. What happens is when KSP gets confused, it will default to the first biome in the list. In this case, that is Olympus Mons. I will take a look at seeing if I can find my safe colors and poke at this.

**Comment by PhineasFreak** [comment]
Worst case scenario: make "lowlands" the default (fallback) biome.

**Comment by RCrockford** [comment]
This should be addressed by #195
